docs/swagger/spec.yml

```diff
@@ -20,8 +20,12 @@ consumes:
 tags:
```

Comment:
`get /buckets/` is missing filtering parameters - they're on `get /buckets/{uuid}/` instead